xml/deploy_cephadm.xml

```diff
@@ -797,6 +797,13 @@ config: OK
      valid, it is a good idea to export its configuration to a file:
     </para>
 <screen>&prompt.smaster;ceph-salt export > cluster.json</screen>
+    <warning>
+      <para>
+       Executing the <command>ceph-salt export</command> includes the
+       SSH private key. If you are concerned about the security implications,
+       then do not execute this command unless necessary.
```

Comment:
instead of "do not execute this command unless necessary" we could say "do not execute this command without taking appropriate precautions" ?